lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
I think it's due to a recent update of Travis CI's default images. @kivikakk fixed it in [her pull request](https://github.com/github/linguist/pull/3735).